src/StripeTests/Infrastructure/EnumDeserializationTest.cs

```diff
@@ -0,0 +1,43 @@
+namespace StripeTests
+{
+    using Newtonsoft.Json;
+    using StripeTests.Infrastructure.TestData;
+    using Xunit;
+
+    public class EnumDeserializationTest : BaseStripeTest
+    {
+        // TODO: find a better place for this test
+        [Fact]
+        public void EnumDecodeUnknownValue()
+        {
+            var json = "{\"enum\": \"TestTwo\"}";
+            TestOptions obj = JsonConvert.DeserializeObject<TestOptions>(json);
+
+            Assert.NotNull(obj);
+            Assert.Equal(TestOptions.TestEnum.TestTwo, obj.Enum);
```

Comment:
Isn't this value supposed to be unknown?

Reply:
The test is badly named -- the value is not "unknown", `TestTwo` merely doesn't define an explicit serialization value. I'll rename the test method.